tests/cypress/support/utils.ts

```diff
@@ -14,3 +16,35 @@ export const isK8sVersion = (version: string) => {
 export const isRancherManagerVersion = (version: string) => {
   return (new RegExp(version)).test(Cypress.env("rancher_version"));
 }
+
+// Run `It()` block based on allowedVersions
```

Comment:
Per my understanding not exclusively to `It()` blocks. Suggest change: `It() / Describe() blocks`

Reply:
I will try and make it describe level version mapping.